instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/tracer/SpanNames.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.api.tracer;
+
+import java.lang.reflect.Method;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+public final class SpanNames {
+  /**
+   * This method is used to generate an acceptable span (operation) name based on a given method
+   * reference. Anonymous classes are named based on their parent.
+   */
+  public static String spanNameForMethod(Method method) {
+    return spanNameForMethod(method.getDeclaringClass(), method.getName());
+  }
+
+  /**
+   * This method is used to generate an acceptable span (operation) name based on a given method
+   * reference. Anonymous classes are named based on their parent.
+   */
+  public static String spanNameForMethod(Class<?> clazz, @Nullable Method method) {
```

Comment:
Not for this PR, just for everyone's thought I feel like this method probably isn't necessary, does / should `clazz` not always be `method.getDeclaringClass()`?

Reply:
yeah. you were right. let't remove this and use `method.getDeclaringClass()`